pkg/schema/openapi.go

```diff
@@ -57,11 +58,15 @@ func (o *OpenAPIDocument) calculateProperties(schemaVal interface{}) *yamlmeta.M
 			mi := yamlmeta.MapItem{Key: i.Key, Value: o.calculateProperties(i.GetValueType())}
 			properties = append(properties, &mi)
 		}
-		property := yamlmeta.Map{Items: []*yamlmeta.MapItem{
-			{Key: typeProp, Value: "object"},
-			{Key: "additionalProperties", Value: false},
-		}}
-
+		var property yamlmeta.Map
+		if typedValue.GetTitle() != "" {
+			item := []*yamlmeta.MapItem{
+				{Key: titleProp, Value: typedValue.GetTitle()},
+			}
+			property.Items = append(property.Items, item...)
```

Comment:
`item` need not be a slice of `yamlmeta.MapItem`, we know we're adding exactly one. That makes declaring an array/slice "accidental complexity."

Let's make it a simple pointer to a map item, instead.

Reply:
Made the changes. The code looks simpler and consistent now.